### ISSUE TITLE
Add image crop feature

### ISSUE THREAD

**Issue by birdkiwi** [linuxmint/xviewer#21]
Sometimes it could be handy

**Comment by sandipkc7** [comment]
I have this image viewer as default for all image viewing. But I had to install another one just for cropping feature. So cropping could be handy sometimes.

**Comment by alextpedro** [comment]
Would love this, too.

**Comment by mikehaertl** [comment]
@JosephMcc Any chance to see this new feature? We already have rotate and mirror, so cropping (and maybe resize?) would complete the toolbox of basic image commands.

**Comment by JosephMcc** [comment]
I'm not sure. We also have pix which offers more features. Personally I'd prefer to see xviewer as a simple image viewing tool and not have a ton of overlap between the two.

**Comment by mikehaertl** [comment]
To be honest I didn't even know about Pix until your comment. As xviewer is the default viewer on Mint, my standard flow is like: double click image, look for "Crop" under "Edit", rant, open Gimp ...

If you look at the mobile sector, most default image viewers there also provide those basic capabilities. So I would argue it's what most users expect nowadays. To be clear: I'm not talking about a full featured image manipulation tool. But the most basic operations should be there to make them easy to find.

**Comment by Thorny-Twig** [comment]
Yes, the CROP tool would be an extremely useful tool to have with Xviewer and was the first thing I looked for when going through menu option after taking/opening a defaulted full desktop screenshot.

**Comment by cave-felem** [comment]
Would be just great to have a crop feature. For me it's simply a very basic thing to have. And as mentioned above, xviewer is the default for Linux Mint...

**Comment by zhanglongqi** [comment]
@birdkiwi  I installed shotwell for this purpose.

**Comment by Julvenzor** [comment]
I second this.

Regards.

**Comment by saukrs** [comment]
I second this request too.

**Comment by Caseyb87** [comment]
Agree with what's been posted. Cropping is a basic feature that fits perfectly fine with a "viewer" application. Would be nice to have added.

**Comment by stefan123t** [comment]
I can only second the resize and crop features, as well as the much needed obfuscate / pixelize tool 
for privacy in many scans, images, etc. which I want / need to send out. 

I use https://apps.gnome.org/en/Obfuscate/ for that purpose at the moment or Gimp Filter > Pixelize with 5-8 pixels size.
I do use Greenshot Editor on Windows for the same purposes as the pixelize tool there is pretty nice.

**Comment by ia-S-on** [comment]
Crop
Flip (x or y)
Resize (By pixel, cm, in, or filesize)
Rotate by XXX.xx Degrees 

!!!!!!!!!!!!!!!
